iceberg/iceberg-handler/src/test/queries/positive/describe_iceberg_table.q

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS ice_t;
+CREATE EXTERNAL TABLE ice_t (i int, s string, ts timestamp, d date) STORED BY ICEBERG;
+
+DROP TABLE IF EXISTS ice_t_transform;
+CREATE EXTERNAL TABLE ice_t_transform (year_field date, month_field date, day_field date, hour_field timestamp, truncate_field string, bucket_field int, identity_field int) PARTITIONED BY SPEC (year(year_field), month(month_field), day(day_field), hour(hour_field), truncate(2, truncate_field), bucket(2, bucket_field), identity_field) STORED BY ICEBERG;
+
+DROP TABLE IF EXISTS ice_t_transform_prop;
+CREATE EXTERNAL TABLE ice_t_transform_prop (id int, year_field date, month_field date, day_field date, hour_field timestamp, truncate_field string, bucket_field int, identity_field int) STORED BY ICEBERG TBLPROPERTIES ('iceberg.mr.table.partition.spec'='{"spec-id":0,"fields":[{"name":"year_field_year","transform":"year","source-id":1,"field-id":1000},{"name":"month_field_month","transform":"month","source-id":2,"field-id":1001},{"name":"day_field_day","transform":"day","source-id":3,"field-id":1002},{"name":"hour_field_hour","transform":"hour","source-id":4,"field-id":1003},{"name":"truncate_field_trunc","transform":"truncate[2]","source-id":5,"field-id":1004},{"name":"bucket_field_bucket","transform":"bucket[2]","source-id":6,"field-id":1005},{"name":"identity_field","transform":"identity","source-id":7,"field-id":1006}]}');
+
+DROP TABLE IF EXISTS ice_t_identity_part;
+CREATE EXTERNAL TABLE ice_t_identity_part (a int) PARTITIONED BY (b string) STORED BY ICEBERG;
+
+DESCRIBE FORMATTED ice_t;
+DESCRIBE FORMATTED ice_t_transform;
+DESCRIBE FORMATTED ice_t_transform_prop;
+DESCRIBE FORMATTED ice_t_identity_part;
+
+SET hive.ddl.output.format=json;
+DESCRIBE EXTENDED ice_t;
```

Comment:
The json output of these are not supposed to show up in the q.out file? Is it because `EXTENDED` is used here or because the output format is json?

Reply:
When the output format is set to `json` the partition information is only available when the `EXTENDED` keyword is used. But in q tests, the output of every `DESCRIBE EXTENDED` command is redacted... :(